common/lib/dependabot/pull_request_creator/message_builder/metadata_presenter.rb

```diff
@@ -0,0 +1,283 @@
+          msg +=
+            begin
+              changelog_lines = changelog_text.split("\n").first(50)
+              changelog_lines = changelog_lines.map { |line| "> #{line}\n" }
+              changelog_lines << truncated_line if changelog_lines.count == 50
+              changelog_lines.join
+            end
+          msg = link_issues(text: msg, dependency: dependency)
+          msg = fix_relative_links(text: msg, base_url: changelog_url)
+          msg = sanitize_template_tags(msg)
+          msg = sanitize_links_and_mentions(msg)
+
+          build_details_tag(summary: "Changelog", body: msg)
+        end
+
+        def upgrade_guide_cascade
+          return "" unless upgrade_guide_url && upgrade_guide_text
+
+          msg = "*Sourced from "\
+                "[#{dependency.display_name}'s upgrade guide]"\
+                "(#{upgrade_guide_url}).*\n\n"
+          msg +=
+            begin
+              upgrade_lines = upgrade_guide_text.split("\n").first(50)
+              upgrade_lines = upgrade_lines.map { |line| "> #{line}\n" }
+              upgrade_lines << truncated_line if upgrade_lines.count == 50
+              upgrade_lines.join
+            end
+          msg = link_issues(text: msg, dependency: dependency)
+          msg = fix_relative_links(text: msg, base_url: upgrade_guide_url)
+          msg = sanitize_template_tags(msg)
+          msg = sanitize_links_and_mentions(msg)
+
+          build_details_tag(summary: "Upgrade guide", body: msg)
+        end
+
+        def commits_cascade
+          return "" unless commits_url && commits
+
+          msg = ""
+
+          commits.reverse.first(10).each do |commit|
+            title = commit[:message].strip.split("\n").first
+            title = title.slice(0..76) + "..." if title && title.length > 80
+            title = title&.gsub(/(?<=[^\w.-])([_*`~])/, '\\1')
+            sha = commit[:sha][0, 7]
+            msg += "- [`#{sha}`](#{commit[:html_url]}) #{title}\n"
+          end
+
+          msg = msg.gsub(/\<.*?\>/) { |tag| "\\#{tag}" }
+
+          msg +=
+            if commits.count > 10
+              "- Additional commits viewable in "\
+              "[compare view](#{commits_url})\n"
+            else
+              "- See full diff in [compare view](#{commits_url})\n"
+            end
+          msg = link_issues(text: msg, dependency: dependency)
+          msg = sanitize_links_and_mentions(msg)
+
+          build_details_tag(summary: "Commits", body: msg)
+        end
+
+        def maintainer_changes_cascade
+          return "" unless maintainer_changes
+
+          build_details_tag(
+            summary: "Maintainer changes",
+            body: sanitize_links_and_mentions(maintainer_changes) + "\n"
+          )
+        end
+
+        def build_details_tag(summary:, body:)
+          # Azure DevOps does not support <details> tag (https://developercommunity.visualstudio.com/content/problem/608769/add-support-for-in-markdown.html)
```

Comment:
Flagging conflict https://github.com/dependabot/dependabot-core/pull/2536 ; `<details>` maybe supported now?

Nothing to do here, just caught in the same morning's inbox.